docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -35,7 +35,7 @@ helpviewer_keywords:
 ---
 # Member access operators (C# reference)
 
-You might use the following operators when you access a type member:
+You can use the following operators when you access a type member:
 
 - [`.` (member access)](#member-access-operator-): to access a member of a namespace or a type
```

Comment:
```suggestion
- [`.` (member access)](#member-access-operator-): to access a member of a namespace or a type.
```

Reply:
No punctuation according to this guideline:
https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation